src/doc/book/patterns.md

```diff
@@ -7,7 +7,7 @@ on a whirlwind tour of all of the things patterns can do!
 [bindings]: variable-bindings.html
 [match]: match.html
 
-A quick refresher: you can match against literals directly, and `_` acts as an
+A quick refresher; you can match against literals directly, and `_` acts as an
```

Comment:
Agreed, it is introducing an idea per [this reference](http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_04.htm)